server/src/inmantals/server.py

```diff
@@ -553,7 +565,10 @@ async def shutdown(self, **kwargs):
         logger.debug("shutdown requested...")
         if self.tmp_project:
             self.tmp_project.cleanup()
-        self.threadpool.shutdown(cancel_futures=True)
+        if CORE_VERSION < version.Version("5.dev"):
+            self.threadpool.shutdown()
+        else:
+            self.threadpool.shutdown(cancel_futures=True)
```

Comment:
The cancel_futures argument doesn't exist for python < 3.9 https://docs.python.org/3.6/library/concurrent.futures.html#concurrent.futures.Executor.shutdown